js/stores/appStore.js

```diff
@@ -568,6 +570,9 @@ const handleAppAction = (action) => {
         if (device.name) {
           appState = appState.setIn(['sync', 'devices', deviceId, 'name'], device.name)
         }
+        if (!hasMainDevice && getSetting(settings.SYNC_DEVICE_NAME) === device.name) {
+          appState = appState.setIn(['sync', 'devices', deviceId, 'mainDevice'], true)
```

Comment:
mainDevice should be added to docs/state.md

Reply:
done in 49113e5b3571ccaabaeeb7d6b69ef415cb137dfe